examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/ComponentMulti.java

```diff
@@ -0,0 +1,28 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        /* Activate the component immediately, also if no one depends on it. */
+        immediate = true,
+        /* You need to implement some class, without any it will not be picked up. */
+        service = { ComponentMulti.class },
+        /*
+         * Set the scope and function(s) properties.
+         * If multiple functions are provided repeat the function property for all of them.
+         */
+        property = { //
+                Constants.COMMAND_SCOPE + "=mycm", //
+                Constants.COMMAND_FUNCTION + "=foo", //
+                Constants.COMMAND_FUNCTION + "=bar" //
+        })
```

Comment:
These properties should not be added as raw Strings, but using DS 1.4 Component Property Type support. If there aren't existing Component Property annotations for Gogo (probably something to raise with the Felix team) then this example could show how to create them.

Reply:
Isn't this the same code as is provided by the enroute FAQ example page?

Reply:
And that page should also be updated (it's not new content). It's important that the examples show new users the nice ways to do these things, not what we had to live with in the past 😃

Reply:
At the moment I am not such familiar with the new stuff of R7.
I will postpone that PR until there is something provided by Felix's Gogo bundles.